### PR TITLE
Smooth if (mu < 0)

### PR DESCRIPTION
**Pull request by vladotomov**
viscosity / default arguments / definition of total rate

### CODE REVIEW

File: laghos.cpp [hunk 1]
```diff
@@ -172,7 +173,7 @@ int main(int argc, char *argv[])
       case 111:
          unit = floor(pow(num_tasks, 1.0 / dim) + 1e-2);
          for (int d = 0; d < dim; d++) { nxyz[d] = unit; }
-         if (dim == 2) { nxyz[2] = 0; }
+         if (dim == 3) { nxyz[2] = 0; }
```
Comment (tzanio) [inline laghos.cpp]: Why this change?

Comment (vladotomov) [reply laghos.cpp]: nxyz[2] is allocated only in 3D.

**Comment by tzanio** [comment]
Looks good, thanks Vladimir!

**Review by tzanio — APPROVED** [review]

**Review by camierjs — COMMENTED** [review]
Good for me, I get all the right results.

**Review by camierjs — APPROVED** [review]
